sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -352,11 +354,13 @@ public virtual async Task<CopyModelOperation> StartCopyModelAsync(string modelId
         /// <summary>
         /// Generate authorization for copying a custom model into the target Form Recognizer resource.
         /// </summary>
-        /// <param name="resourceId">Azure Resource Id of the target Form Recognizer resource where the model will be copied to.</param>
-        /// <param name="resourceRegion">Location of the target Form Recognizer resource</param>
+        /// <param name="resourceId">Azure Resource Id of the target Form Recognizer resource where the model will be copied to.
+        /// This information can be found in the Properties section of the Form Recognizer resource in the Azure Portal.</param>
+        /// <param name="resourceRegion">Location of the target Form Recognizer resource.
+        /// This information can be found in the Properties section of the Form Recognizer resource in the Azure Portal.</param>
```

Comment:
@kristapratico couldn't find the link for all the available regions. Do you have it?

Reply:
https://azure.microsoft.com/global-infrastructure/services/?products=cognitive-services

Reply:
Thank you.

Reply:
`resourceRegion` should be the `Location`  from the Keys section? Or the format `West US 2` also works?

Reply:
Just saw this. sorry!!hmm yeah, better to say the Locaiton in Keys. I will update that in a future PR
thanks for pointing that out Jeremy!

Reply:
PR https://github.com/Azure/azure-sdk-for-net/pull/13096